sabnzbd/utils/file_extension.py

```diff
@@ -227,7 +226,6 @@
     "part",
     "pdf",
     "pem",
-    "php",
```

Comment:
Php seems like one we shouldn't rename?

Reply:
My reasoning: "PHP in a download is very unlikely. And if it's there, it's probably malevolent."

I checked my own drive, and there are 3 .php files. Two look certainly not php, one probably not. Maybe extension-renaming gave them the php extension. Reason more to remove .php from the list?